src/game-timer/game-timer.html

```diff
@@ -0,0 +1,83 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
```

Comment:
The code looks good, and I think after this point we can add the win condition and we are done.